modules/core/shared/src/test/scala/eu/timepit/refined/ImpureValidateSpec.scala

```diff
@@ -0,0 +1,62 @@
+package eu.timepit.refined
+
+import eu.timepit.refined.TestUtils._
+import eu.timepit.refined.boolean._
+import eu.timepit.refined.collection._
+import eu.timepit.refined.impure.NonNull
+import eu.timepit.refined.numeric._
+import eu.timepit.refined.string._
+import org.scalacheck.Prop._
+import org.scalacheck.Properties
+
+class ImpureValidateSpec extends Properties("ImpureValidate") {
+
+  property("NonNull safe for String") = secure {
+    import eu.timepit.refined.collection.NonEmpty
```

Comment:
I believe `NonEmpty` is already being imported on line 5 -`import eu.timepit.refined.collection._`?

Reply:
You are correct.